src/fmu/dataio/dataio.py

```diff
@@ -690,6 +690,9 @@ def _update_check_settings(self, newsettings: dict) -> None:
         self._validate_and_establish_fmucontext()
         self._rootpath = self._establish_rootpath()
 
+        self._classification = self._get_classification()
+        self._rep_include = self._get_rep_include()
```

Comment:
Moved into the `update_check_settings()` as it is only needed to run if any of `classification/rep_include/access_ssdl` is coming in through `kwargs.` And hence it can be removed together with the entire  `update_check_settings()` method when the deprecation period has ended.